json_schemas/suppliers.json

```diff
@@ -26,7 +26,9 @@
     },
     "dunsNumber": {
       "pattern": "^[0-9]+$",
-      "type": "string"
+      "type": "string",
+      "minLength": 9,
+      "maxLength": 9
```

Comment:
Are there any existing suppliers with a DUNS number of the wrong length? Suppliers can't change this value once they've created an account, so they won't be able to edit any other days if their DUNS doesn't validate.